spyder/plugins/variableexplorer/widgets/namespacebrowser.py

```diff
@@ -34,7 +34,9 @@
 from spyder.utils.misc import getcwd_or_home, remove_backslashes
 from spyder.utils.programs import is_module_installed
 from spyder.utils.qthelpers import (add_actions, create_action,
-                                    create_toolbutton, create_plugin_layout,
+                                    create_toolbutton,
+                                    create_plugin_layout,
+                                    create_waitspinner,

```

Comment:
These lines are one space to the right with respect to the left parenthesis above. Please fix it.